Makefile

```diff
@@ -1,37 +1,62 @@
 # Extremely simple HTTP server that responds on port 8000 with a hello message.
 
-DOCKERHUB_ID:=ibmosquito
+DOCKER_HUB_ID ?= ibmosquito
 NAME:="web-hello-go"
-VERSION:="1.0.0"
+PATTERN_NAME:="pattern-web-helloworld-go"
+SERVICE_VERSION:="1.0.0"
+ARCH:="amd64"
+
+CONTAINER_CREDS:=
 
 default: build run
 
 build:
-	docker build -t $(DOCKERHUB_ID)/$(NAME):$(VERSION) .
+	docker build -t $(DOCKERHUB_ID)/$(SERVICE_NAME):$(SERVICE_VERSION) .
 
 dev: stop build
 	docker run -it -v `pwd`:/outside \
-          --name ${NAME} \
+          --name ${SERVICE_NAME} \
           -p 8000:8000 \
-          $(DOCKERHUB_ID)/$(NAME):$(VERSION) /bin/bash
+		  $(DOCKERHUB_ID)/$(SERVICE_NAME):$(SERVICE_VERSION) /bin/bash
 
 run: stop
 	docker run -d \
-          --name ${NAME} \
+          --name ${SERVICE_NAME} \
           --restart unless-stopped \
           -p 8000:8000 \
-          $(DOCKERHUB_ID)/$(NAME):$(VERSION)
+           $(DOCKERHUB_ID)/$(SERVICE_NAME):$(SERVICE_VERSION)
 
 test:
 	@curl -sS http://127.0.0.1:8000
 
 push:
-	docker push $(DOCKERHUB_ID)/$(NAME):$(VERSION) 
-
+	docker push $(DOCKERHUB_ID)/$(SERVICE_NAME):$(SERVICE_VERSION) 
+
+publish-service:
+	    @ARCH=$(ARCH) \
```

Comment:
I believe the issue you were seeing has to do with tab/spaces. Makefiles can be pretty picky about that. Make sure all the lines below each target are one tab in. (I believe it's tabs it wants)